.github/workflows/golint.yaml

```diff
@@ -4,12 +4,12 @@ jobs:
   lint:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v3
-      with:
-        fetch-depth: 1
-    - uses: WillAbides/setup-go-faster@v1.7.0
-      with:
-        go-version: "1.19.x"
-    - uses: dominikh/staticcheck-action@v1.2.0
-      with:
-        version: "2022.1.1"
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 1
+      - uses: WillAbides/setup-go-faster@v1.7.0
+        with:
+          go-version: "1.19.x"
+      - uses: dominikh/staticcheck-action@v1.3.0
+        with:
+          version: "2023.1.6"
```

Comment:
Can we revert the extra spaces added?
Otherwise I can merge this PR to a feature branch and clean it up become bringing it to `main` : )